ecs-cli/modules/clients/aws/cloudformation/client.go

```diff
@@ -151,6 +152,12 @@ func (c *cloudformationClient) DeleteStack(stackName string) error {
 	return err
 }
 
+func (c *cloudformationClient) DescribeStacks(stackName string) (*cloudformation.DescribeStacksOutput, error) {
```

Comment:
looks like Travis is failing b/c it can't find this. Do you need to regen the mocks?

Reply:
also, this public method needs a comment to avoiding style warnings